src/connection-manager/agreement.ts

```diff
@@ -102,6 +108,16 @@ async function updateDailyAgreement(
   })
 }
 
+/**
+ * Detect the ledger before a flagged ledger, which will contain server version information.
+ *
+ * @param ledger_index - Index of current ledger.
+ * @returns Boolean.
+ */
+function preceedFlagLedger(ledger_index: string): boolean {
```

Comment:
nit, would prefer if this was `preceedsFlagLedger()`

Reply:
This should involve some verb like `is` or `does` - maybe `isPreceedingFlagLedger`? To make it clear that it's returning a boolean